regression/cbmc-java/stack_var1/README

```diff
@@ -0,0 +1,2 @@
+This regression test is created from the .j file with the jasmin assembler.
+https://github.com/Sable/jasmin
```

Comment:
Would it be possible to add as many details as possible, such as the precise command line? This is information that you've now got right at hand, but it will be much more work to reproduce later on. Memory is weak, git is strong!